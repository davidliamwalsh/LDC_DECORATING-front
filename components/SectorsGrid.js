import { Component } from 'react'
import Link from 'next/link'

class SectorsGrid extends Component {
  render () {
    return (
      <div className="c-service-sector--grid">
        <div className="c-service-sector--grid__item" data-aos="fade-right">
          <div className="c-service-sector--grid__item--title">
            <h2>Retail &amp; Shop Fitting</h2>
            <hr className="o-line" />
          </div>
          <Link href='/sectors/retail-shops'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          <div className="c-service-sector--grid__item--footer">
            <h4>MAKE YOUR BUSINESS STAND OUT</h4>
          </div>
        </div>

        <div className="c-service-sector--grid__item" data-aos="fade-left">
          <div className="c-service-sector--grid__item--title">
            <h2>Factories &amp; Industrial</h2>
            <hr className="o-line" />
          </div>
          <Link href='/sectors/factories-industrial'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          <div className="c-service-sector--grid__item--footer">
            <h4>INTERIOR &amp; EXTERIOR REFURBISHMENTS</h4>
          </div>
        </div>

        <div className="c-service-sector--grid__item" data-aos="fade-right">
          <div className="c-service-sector--grid__item--title">
            <h2>Schools</h2>
            <hr className="o-line" />
          </div>
          <Link href='/sectors/schools'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          <div className="c-service-sector--grid__item--footer">
            <h4>EDUCATION REDECORATION SPECIALISTS</h4>
          </div>
        </div>

        <div className="c-service-sector--grid__item" data-aos="fade-left">
          <div className="c-service-sector--grid__item--title">
            <h2>Office Space</h2>
            <hr className="o-line" />
          </div>
          <Link href='/sectors/offices'>
            <a><div className="c-service-sector--grid__item--image c-service-sector--grid__item--wall-coverings" /></a>
          </Link>
          <div className="c-service-sector--grid__item--footer">
            <h4>OFFICE REDECORATION SERVICES</h4>
          </div>
        </div>
      </div>
    )
  }
}

export default SectorsGrid