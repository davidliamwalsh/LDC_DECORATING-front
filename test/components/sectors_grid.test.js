import SectorsGrid from '../../components/SectorsGrid'

describe('<SectorsGrid />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <SectorsGrid />
    )

    expect(component).toMatchSnapshot()
  })
})