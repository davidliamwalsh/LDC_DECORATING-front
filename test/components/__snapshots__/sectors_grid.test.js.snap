// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectorsGrid /> it renders correctly 1`] = `ShallowWrapper {}`;
